internal/databases/postgres/backup_fetch_handler.go

```diff
@@ -62,9 +62,8 @@ func chooseTablespaceSpecification(sentinelDtoSpec, spec *TablespaceSpec) *Table
 
 // TODO : unit tests
 // deltaFetchRecursion function composes Backup object and recursively searches for necessary base backup
-func deltaFetchRecursionOld(backupName string, folder storage.Folder, dbDataDirectory string,
+func deltaFetchRecursionOld(backup Backup, folder storage.Folder, dbDataDirectory string,
```

Comment:
This change is not related to the files metadata, but helps to avoid downloading it twice.